### PR TITLE
[build-compare] Ignore parts of NSS checksum files. JB#58644

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by vigejolla** [comment]
Wrong bug ID?

**Comment by martyone** [comment]
Hm, I wonder where I saw that number - fixed.

**Review by vigejolla — APPROVED** [review]
The PR title still has wrong bug ID. Otherwise LGTM.

File: rpm/build-compare.spec [hunk 1]
```diff
@@ -22,6 +22,7 @@ Release:        1
 Source0:        %{name}-%{version}.tar.gz
 Patch0:         0001-OBS-project-name-in-RPM-meta-data-causes-problems-in.patch
 Patch1:         0002-fix-compatibility-with-older-sed.patch
+Patch1:         0003-ignore-parts-of-NSS-checksum-files.patch
```
Comment (martyone) [inline rpm/build-compare.spec:25]: Oops.. fixed with https://github.com/sailfishos/build-compare/commit/dc32a1c05d68abb3bffee8e0d57459a8eed95b71

**Pull request merged**